internal/uvm/create_lcow.go

```diff
@@ -199,23 +199,6 @@ func CreateLCOW(opts *OptionsLCOW) (_ *UtilityVM, err error) {
 		}
 	}
 
-	if !opts.KernelDirect {
-		doc.VirtualMachine.Devices.VirtualSmb = &hcsschema.VirtualSmb{
-			Shares: []hcsschema.VirtualSmbShare{
-				{
-					Name: "os",
-					Path: opts.BootFilesPath,
-					Options: &hcsschema.VirtualSmbShareOptions{
```

Comment:
We had this in place because of options. Are you saying we dont need any of these options?

Reply:
Yes, that’s what I’m saying.